lib/core/presentation/pages/chat/chat_message/widgets/message_item/image_message_widget.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:app/core/presentation/widgets/chat/mxc_image.dart';
+import 'package:app/core/presentation/widgets/hero_image_viewer_widget.dart';
+import 'package:app/core/presentation/widgets/loading_widget.dart';
+import 'package:flutter/material.dart';
+
+import 'package:flutter_blurhash/flutter_blurhash.dart';
+import 'package:matrix/matrix.dart';
+
+class ImageMessage extends StatelessWidget {
+  final Event event;
+  final BoxFit fit;
+  final bool maxSize;
+  final Color? backgroundColor;
+  final bool thumbnailOnly;
+  final bool animated;
+  final double width;
+  final double height;
+  final void Function()? onTap;
+
+  const ImageMessage(
+    this.event, {
+    this.maxSize = true,
+    this.backgroundColor,
+    this.fit = BoxFit.cover,
+    this.thumbnailOnly = true,
+    this.width = 400,
+    this.height = 300,
+    this.animated = false,
+    this.onTap,
+    Key? key,
+  }) : super(key: key);
+
+  Widget _buildPlaceholder(BuildContext context) {
+    if (event.messageType == MessageTypes.Sticker) {
+      return const Center(
+        child: CircularProgressIndicator.adaptive(),
+      );
+    }
+    final String blurHashString = event.infoMap['xyz.amorgan.blurhash'] is String
+        ? event.infoMap['xyz.amorgan.blurhash']
+        : 'LEHV6nWB2yk8pyo0adR*.7kCMdnj';
```

Comment:
Do we need define some blur utility for this

Reply:
I think no need, maybe can name it as defaultBlurhash